frontend/views/containers/access/SignupForm.vue

```diff
@@ -102,8 +106,12 @@ export default ({
       form: {
         username: {
           [L('A username is required.')]: required,
-          [L('A username cannot contain spaces.')]: nonWhitespace,
+          [L('A username cannot contain white space.')]: nonWhitespace,
```

Comment:
I am pretty sure that `white space` should be spelled `whitespace` as it's a single thing, and "white space" makes it sound like the space is the color white or something. Even though it might trigger some spellchecks, it seems like they're just not updated: https://en.wikipedia.org/wiki/Whitespace_character